### PR TITLE
Move json methods to shared util module

### PR DESCRIPTION
**Pull request by chuckhays**
Moves json methods to util module so we can use them in apprtc.py. Adds a couple of unit tests for the util methods.

Sending now as a separate PR to avoid merge conflicts with multiple people working in this code.

@tkchin @jiayliu

### CODE REVIEW

File: samples/web/content/apprtc/util_test.py [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+    testObject = {
+      "a": False,
+      "b": "str",
+      "c": None,
+      "d": {},
+      "e": [1, 2, "3"],
+      "f": [],
+      "g": {'A': 1}
+    }
+    self.assertEqual(
```
Comment (jiayliu) [inline samples/web/content/apprtc/util_test.py]: Also add a test case for a key not in the dict?

**Comment by jiayliu** [comment]
lgtm with nit

File: samples/web/content/apprtc/util_test.py (hunk 1 above)
Comment (chuckhays) [inline samples/web/content/apprtc/util_test.py]: Added a test for missing key and passing None as a message, which could happen if the json is invalid.

**Comment by chuckhays** [comment]
Added 2 test cases and made a resulting change to the method, PTAL.

**Pull request merged**